packages/flatbread/src/cli/runner.ts

```diff
@@ -29,8 +29,10 @@ export default function orchestrateProcesses({
   let serverModulePath = 'node_modules/flatbread/dist/graphql/server.js';
 
   process.cwd();
-  const gql = fork(resolve(process.cwd(), serverModulePath), [], {
+  const gql = fork(resolve(process.cwd(), serverModulePath), [''], {
```

Comment:
fancy, we might want to think about hot reloading in the future if the config changes

Reply:
absolutely - I've had that in mind but haven't cut a ticket for it yet

Reply:
tracked here https://github.com/tonyketcham/flatbread/issues/64